tfhe/docs/how_to/rayon_crate.md

```diff
@@ -0,0 +1,180 @@
+# Making rayon and tfhe-rs work together
+
+[rayon](https://crates.io/crates/rayon) is a popular create to easily write multi-threaded code in Rust.
+
+It is possible to use rayon to write multi-threaded tfhe-rs code. However due to internal details of `rayon` and
```

Comment:
I think the "brand name" is TFHE-rs ? also one is between backticks but not the other, don't know what was chosen for that, I would say backticks here so that it stands out the same way rayon will